parent/pom.xml

```diff
@@ -229,7 +229,7 @@
         <cxf.woodstox.core.version>6.2.7</cxf.woodstox.core.version>
         <cxf.woodstox.stax2-api.version>4.2.1</cxf.woodstox.stax2-api.version>
         <cxf.wsdl4j.version>1.6.3</cxf.wsdl4j.version>
-        <cxf.jakarta.wss4j.version>3.0.0-SNAPSHOT</cxf.jakarta.wss4j.version>
+        <cxf.jakarta.wss4j.version>2.4.1</cxf.jakarta.wss4j.version>
```

Comment:
@jimma I reverted that for now (there are artifacts we use but non-existing in 3.0.0-SNAPSHOT) so we could hopefully address that as part of https://github.com/apache/cxf/pull/927

Reply:
@reta  OK. After the this security failure is fixed, we can ask for a wss4j jakarta release to include it in CXF.

Reply:
👍